packages/block-library/src/search/edit.js

```diff
@@ -125,8 +125,10 @@ export default function SearchEdit( {
 	}
 
 	const colorProps = useColorProps( attributes );
-	const fluidTypographySettings = useSetting( 'typography.fluid' );
-	const layout = useSetting( 'layout' );
+	const [ fluidTypographySettings, layout ] = useSettings(
+		'typography.fluid',
+		'layout'
```

Comment:
Should we get `layout.wideSize` directly here since that's the only one we need?

Reply:
Maybe, but I'll keep the existing `'layout'` retrieval. The codebase consistently uses `'layout'`, never its subfields.